src/helm/benchmark/run_specs_openchatkit.py

```diff
@@ -0,0 +1,1640 @@
+import itertools
+from typing import Any, Callable, List, Dict, Optional, Set
+
+from helm.common.hierarchical_logger import hlog, htrack
+from helm.common.object_spec import ObjectSpec
+from helm.benchmark.adaptation.adapters.adapter_factory import (
+    ADAPT_LANGUAGE_MODELING,
+    ADAPT_MULTIPLE_CHOICE_JOINT,
+    ADAPT_MULTIPLE_CHOICE_SEPARATE_ORIGINAL,
+    ADAPT_MULTIPLE_CHOICE_SEPARATE_CALIBRATED,
+    ADAPT_GENERATION,
+    ADAPT_RANKING_BINARY,
+)
+from helm.benchmark.adaptation.adapters.binary_ranking_adapter import BinaryRankingAdapter
+from helm.benchmark.adaptation.adapter_spec import AdapterSpec
+from .metrics.metric import MetricSpec
+from .run_expander import RUN_EXPANDERS, GlobalPrefixRunExpander, StopRunExpander
+from .runner import RunSpec
+from .scenarios.lex_glue_scenario import (
+    get_lex_glue_max_train_instances,
+    get_lex_glue_instructions,
+    get_lex_glue_max_tokens,
+)
+from .scenarios.scenario import ScenarioSpec
+from .scenarios.big_bench_scenario import BIGBenchScenario
+from .scenarios.msmarco_scenario import MSMARCOScenario
+from .scenarios.numeracy_scenario import get_numeracy_adapter_spec, RELTYPE_INFO
+from .scenarios.copyright_scenario import datatag2hash_code
+from .scenarios.raft_scenario import get_raft_instructions
+from .scenarios.lextreme_scenario import (
+    get_lextreme_instructions,
+    get_lextreme_max_train_instances,
+    get_lextreme_max_tokens,
+)
+from helm.proxy.models import get_model, NO_NEWLINES_TAG, NLG_PREFIX_TAG
+from helm.common.general import singleton
+
+
+############################################################
+# Prototypical adapter specs
+
+
+def format_instructions(instructions: str) -> str:
+    if len(instructions) > 0:
+        instructions = "<human>: " + instructions + "\n" + "<bot>: Sure, I understand."
+    return instructions
+
+
+def get_multiple_choice_joint_adapter_spec(
+    instructions: str, input_noun: Optional[str], output_noun: str, max_train_instances: int = 5, **kwargs
+) -> AdapterSpec:
+    """
+    [instructions]
+
+    [input_noun]: [input]
+    [reference_1]
+    ...
+    [reference_k]
+    [output_noun]: [output]
+
+    [input_noun]: [input]
+    [reference_1]
+    ...
+    [reference_k]
+    [output_noun]:
+    """
+    return AdapterSpec(
+        method=ADAPT_MULTIPLE_CHOICE_JOINT,
+        instructions=format_instructions(instructions),
+        input_prefix=f"<human>: {input_noun}: " if input_noun is not None else f"<human>: ",
```

Comment:
Do we absolutely need `<human>` and `<bot>` tags, or can we just make it work with the usual HELM prompts (we assume every model has the same prompts)?